stateful/es-data-stateful.yaml

```diff
@@ -53,13 +53,21 @@ spec:
           limits:
             cpu: 1
         ports:
+        - containerPort: 9200
+          name: http
         - containerPort: 9300
           name: transport
         livenessProbe:
           tcpSocket:
             port: transport
           initialDelaySeconds: 20
           periodSeconds: 10
+        readinessProbe:
+          httpGet:
+            path: /_cluster/health
```

Comment:
Again, it's clear you didn't even try this because HTTP_ENABLE is still false.

Reply:
Thanks, yeah, I was diffing from the local changes I had made in another repo, and I missed this one. Sorry about that.